server/modules/analysis/manager/processingChainManager.js

```diff
@@ -12,22 +14,46 @@ import * as ProcessingChainRepository from '../repository/processingChainReposit
 import * as ProcessingStepRepository from '../repository/processingStepRepository'
 import * as ProcessingStepCalculationRepository from '../repository/processingStepCalculationRepository'
 
-// ====== CREATE - Chain
+// ====== Chain
 
-export const createChain = async (user, surveyId, cycle, client = db) =>
+const _createChain = async (user, surveyId, processingChain, client) =>
   await client.tx(async t => {
-    const processingChain = await ProcessingChainRepository.insertChain(surveyId, cycle, t)
+    const processingChainDb = await ProcessingChainRepository.insertChain(surveyId, processingChain, t)
     await ActivityLogRepository.insert(
       user,
       surveyId,
       ActivityLog.type.processingChainCreate,
-      processingChain,
+      processingChainDb,
       false,
       t,
     )
     return ProcessingChain.getUuid(processingChain)
   })
 
+const _updateChainProps = async (user, surveyId, chain, chainDb, client) => {
+  const propsPost = ProcessingChain.getProps(chain)
```

Comment:
I wouldn't call it "propsPost" in the Manager, I wouldn't mention "post" at all here.
What about propsNew and propsOld?